internal/dev_server/adapters/context.go

```diff
@@ -0,0 +1,12 @@
+package adapters
+
+import (
+	"context"
+	ldapi "github.com/launchdarkly/api-client-go/v14"
+)
+
+func WithLdApi(ctx context.Context, client ldapi.APIClient, streamingUrl string) context.Context {
```

Comment:
```suggestion
func WithApiAndSdk(ctx context.Context, client ldapi.APIClient, streamingUrl string) context.Context {
```

Otherwise I know I'm gonna get confused with `WithApi`